Source/WebKit/NetworkProcess/storage/NetworkStorageManager.cpp

```diff
@@ -543,6 +550,7 @@ OriginStorageManager& NetworkStorageManager::originStorageManager(const WebCore:
                 quota = m_originQuotaRatio.value() * volumeCapacity.value();
                 increaseQuotaFunction = { };
             }
+            standardReportedQuota *= *m_originQuotaRatio;
```

Comment:
I do not understand this change, how is it possible that a reported quota (in GB) is assigned a quota ratio (value without any unit).

Reply:
> Can we add a test?
> If the test is too long due to writing large files (somehow a good sign for the mitigation), how about adding a test API to lower the initial standardReportedQuota?

Sure, I can try adding a test with test SPI.

Reply:
Oh sorry, I did not see the `*=`...